core/templates/pages/contributor-dashboard-admin-page/contributor-dashboard-tables/contributor-admin-stats-table.component.spec.ts

```diff
@@ -553,4 +634,152 @@ describe('Contributor stats component', () => {
 
     expect(modalSpy).toHaveBeenCalledWith(CdAdminTranslationRoleEditorModal);
   }));
+
+  describe('when displaying the stats table', () => {
+    it('should return the translation submitter stats attributes correctly',
+      fakeAsync(() => {
+        expect(component.getContributorAdminStatsTableAttributes(
+          translationSubmitterStats)).toEqual([
+          {
+            key: 'Date Joined',
+            value: 'firstcontributiondate'
+          },
+          {
+            key: 'Submitted Translations',
+            value: '3 cards, 4 words'
+          },
+          {
+            key: 'Accepted Translations',
+            value: '5 cards (6 without edits), 7 words'
+          },
+          {
+            key: 'Rejected Translations',
+            value: '8 cards, 9 words'
+          },
+          {
+            key: 'Active Topics',
+            value: 'topic1, topic2'
+          },
+        ]);
+      }));
+    it('should return the translation reviewer stats attributes correctly',
```

Comment:
@stevenjgodfrey Could you please leave a blank line between successive it() statements? It helps make things a bit easier to read. Thanks!

Reply:
Done